app/script/event/EventRepository.coffee

```diff
@@ -355,7 +357,8 @@ class z.event.EventRepository
         return @cryptography_repository.decrypt_event event
         .catch (decrypt_error) =>
           # Get error information
-          error_code = decrypt_error.code or 999
+          decrypt_error.code = UNKNOWN_DECRYPT_ERROR_CODE if not decrypt_error.code
```

Comment:
because the `decrypt_error.code` could be used further down the line as well somewhere..